app/models/notifications/NotificationFrontendModel.scala

```diff
@@ -28,8 +28,9 @@ final case class NotificationFrontendModel(
   ucrBlocks: Seq[UcrBlock],
   masterRoe: Option[String],
   masterSoe: Option[String],
+  crcCode: Option[String],
```

Comment:
You changed the order of arguments. Can you confirm this? Both of the arguments has the same type `Option[String]` so in places where we can use it it'll contains wrong data.

Reply:
Double checked, run tests. Everything looks fine and working.